src/source/PeerConnection/PeerConnection.c

```diff
@@ -1472,20 +1473,39 @@ STATUS addTransceiver(PRtcPeerConnection pPeerConnection, PRtcMediaStreamTrack p
     UINT32 clockRate = 0;
     UINT32 ssrc = (UINT32) RAND(), rtxSsrc = (UINT32) RAND();
     RTC_RTP_TRANSCEIVER_DIRECTION direction = RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV;
+    DOUBLE rollingBufferDurationSec = DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS;
+    DOUBLE rollingBufferBitratebps = DEFAULT_EXPECTED_VIDEO_BIT_RATE;
     RtcMediaStreamTrack videoTrack;
+
+    CHK(pKvsPeerConnection != NULL, STATUS_NULL_ARG);
+
+    // Set defaults for audio and video track. This can be modified to values set up with pRtcRtpTransceiverInit
+    // if so.
```

Comment:
Not fully understanding this comment's wording. Is it the values of the defaults that can be modified or the variables that they are assigned to? And what is the "if so" referring to?

Reply:
Oops. I should have removed that line.